### ISSUE TITLE
providing additional printer columns for each chaos

### ISSUE THREAD

**Issue by STRRL** [chaos-mesh/chaos-mesh#2440]
## Feature Request

**Is your feature request related to a problem? Please describe:**

No.

**Describe the feature you'd like:**

providing additional printer columns for each type of Chaos.

When I execute `kubectl get podchaos` it will only give me names, I think we could give more useful information with `kubectl get` 

**Describe alternatives you've considered:**

No alternatives yet.

**Teachability, Documentation, Adoption, Migration Strategy:**

Which fields should be shown with `kubectl get` is not decided, and maybe we could not provide some "calculation logic" like Pod's READY column (like `0/1`, `<ready containers>/<all containers>`) with only JSONPath

reference: 
- https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#additional-printer-columns
- https://book.kubebuilder.io/reference/generating-crd.html?highlight=AdditionalPrinterColumns#additional-printer-columns

**Comment by ReigenDing** [comment]
Hi, this issue is not clear, right? I have tried this display column.
```sh
$ kubectl get podchaos.chaos-mesh.org/chaos-test
NAME         MODE
chaos-test   one
```
Actually, I do not know what is desired to display.

**Comment by STRRL** [comment]
> Hi, this issue is not clear, right? I have tried this display column.
> 
> ```shell
> $ kubectl get podchaos.chaos-mesh.org/chaos-test
> NAME         MODE
> chaos-test   one
> ```
> 
> Actually, I do not know what is desired to display.

Yes, the "desired fields to display" is not determined now. But I think there are some common meaningful fields, like `action` and `duration`.

I think we could add these columns at first. Then let us discuss the customized display for each type of chaos. :D

**Comment by ReigenDing** [comment]
Ok， I will add `action` and `duration` first.

**Comment by STRRL** [comment]
closed by #2526
